packages/ag-charts-community/src/chart/themes/chartTheme.ts

```diff
@@ -399,6 +398,25 @@ export class ChartTheme {
                 },
             },
         },
+        bubble: {
+            ...ChartTheme.getChartDefaults(),
+            axes: this.cartesianAxisDefault,
+            series: {
+                ...ChartTheme.getScatterSeriesDefaults(),
+                marker: {
+                    ...ChartTheme.getCartesianSeriesMarkerDefaults(),
+                    maxSize: 30,
+                },
+                label: {
+                    enabled: false,
+                    fontStyle: undefined,
+                    fontWeight: undefined,
+                    fontSize: 12,
+                    fontFamily: DEFAULT_FONT_FAMILY as unknown as string,
```

Comment:
`as unknown as string` seems redundant as of [19f5a4422](https://github.com/ag-grid/ag-charts/commit/19f5a4422542460507d9d3f0d6e3b03e1d1f7b5e)

Reply:
Fixed - thanks!